### ISSUE TITLE
Default organization when adding items

### ISSUE THREAD

**Issue by sgronlund** [UTNkar/ordsys#410]
In cases where there only exists one organization, it can be quite cumbersome to always need to check the org field when adding menu items quickly.

Would be nice to get the org to default to the only existing value

**Comment by dakaza98** [comment]
I assume you refer to when adding menu items in the admin pages? An even better way would be to select the organization that the user belongs to since most of the time this is what users will do

**Comment by sgronlund** [comment]
> I assume you refer to when adding menu items in the admin pages? An even better way would be to select the organization that the user belongs to since most of the time this is what users will do

Yes, exactly that would be a better solution rather than fetching all organizations.

**Comment by albinantti** [comment]
Users can only belong to a single organisation. I agree it's cumbersome but this is only an issue for superusers, all other users don't even see the field from what I remember as they can only create items for their own organisation

**Comment by dakaza98** [comment]
> Users can only belong to a single organisation. I agree it's cumbersome but this is only an issue for superusers, all other users don't even see the field from what I remember as they can only create items for their own organisation

Can confirm this! I just tested it and it is only a superuser issue

**Comment by sgronlund** [comment]
Ah, makes sense! Then this would be even lower prio, e.g. nice-to-have label
